src/lucky/param_serializable.cr

```diff
@@ -0,0 +1,177 @@
+module Lucky
+  annotation ParamField
+  end
```

Comment:
I think this won't be used anymore. It seems you can't read an annotation on a macro directly. It would need to be attached to the instance variable or something.